### PR TITLE
Fix for update the program.cs file within the Wasm prj

### PR DESCRIPTION
**Pull request by TomWalkerCodes**
Fix for updating the program.cs file within the Wasm project.

### CODE REVIEW

File: uno-bootcamp/modules/04-Create-rich-responsive-UIs/README.md
Comment (michael-hawker) [inline uno-bootcamp/modules/04-Create-rich-responsive-UIs/README.md:209]:
```suggestion
[docs-visualboundspadding]: https://github.com/unoplatform/uno/blob/master/doc/articles/features/VisibleBoundsPadding.md
```

They just moved this article, so the original link is the correct one.

File: uno-bootcamp/modules/04-Create-rich-responsive-UIs/README.md
Comment (michael-hawker) [inline uno-bootcamp/modules/04-Create-rich-responsive-UIs/README.md:126]:
```suggestion
By adding the `FeatureConfiguration.UIElement.AssignDOMXamlName = true;` into the _Program.cs_ file of your WebAssembly application (Wasm head), you will see the XAML names directly in the DOM by using the browser F12 tools.  Make sure to update the name spaces to include `using Uno.UI`;
```

**Review by michael-hawker — CHANGES_REQUESTED** [review]

**Comment by ghuntley** [comment]
Cheers Tom, I don't have write access to your branch so I'll fix the reviewer comments up after merging.

**Pull request merged**
